### DISCUSSION TITLE
Player-Specific Effects

### DISCUSSION THREAD

**Discussion by nan-value** [wwmm/easyeffects#3169] [Ideas]
Player-specific effects would be Awesome! Imagine you have your EQ and Expander set for your Spotify/Music player whilst simultaneously running Noise Reduction on your Discord Call. 

For Inputs this would be awesome too because you could set your discord call to sound like normal for your teammates but for in-game comms you could do fun stuff like reverb and bass-loudener to have fun with your teammates without annoying your VC Buddies

**Comment by wwmm** [comment]
Something like this was requested a long time ago #899. There is no doubt it would be nice to have. The problem is that this requires a completely different application design.
